core-bundle/src/Twig/Loader/ContaoFilesystemLoaderWarmer.php

```diff
@@ -71,6 +71,8 @@ public function warmUp(string $cacheDir = null): array
             $this->writeIdeAutoCompletionMapping($cacheDir ?? $this->cacheDir);
         }
 
+        $this->loader->clear();
```

Comment:
I don't think this is correct. This would drop all the paths that were added above and would only rely on the internal cache implementation of the filesystem loader.

Here is an alternative approach: #6868